random.txt

```diff
@@ -0,0 +1,441 @@
/www.malwares.com/report/file?hash=0096e1c33bb54494b7be0857302a3298c621cb629d2a44f043488430dd0abd3b', 'tags': ['malware', 'emotet']}, 'type': 'comment', 'id': 'f-0096e1c33bb54494b7be0857302a3298c621cb629d2a44f043488430dd0abd3b-67d03244', 'links': {'self': 'https://www.virustotal.com/api/v3/comments/f-0096e1c33bb54494b7be0857302a3298c621cb629d2a44f043488430dd0abd3b-67d03244'}}, {'attributes': {'date': 1667602351, 'text': '#malware #Smoke Loader\n[b]VT Collection:[/b] https://www.virustotal.com/gui/collection/3e3a2e51f5f354243f87aa3579e7b0d7206c9f8ce3558d9a1b5f8e6fc3fad7da\n[b]Reported in:[/b]\nHatching Triage: https://tria.ge/221104-y1j46safd6\nIntezer: https://analyze.intezer.com/analyses/4cea82d5-cc21-4fcf-b746-1802f4b31cb0\nHybrid Analysis: https://www.hybrid-analysis.com/sample/d1cdab058056e0e4cbf2a08851d493d9f46d1d36e65f7b284d2ecc3558e80660\nJOESandbox: https://www.joesandbox.com/analysis/738404/0/html\nFileScan.IO: https://www.filescan.io/reports/d1cdab058056e0e4cbf2a08851d493d9f46d1d36e65f7b284d2ecc3558e80660/d7b6b626-e2b8-4a97-8ba0-f3fa7e788687/overview\nYOMI: https://yomi.yoroi.company/report/63657265317bd2f2ac8febfd/63657265317bd2f2ac8febfe/overview\nmalwares: https://www.malwares.com/report/file?hash=d1cdab058056e0e4cbf2a08851d493d9f46d1d36e65f7b284d2ecc3558e80660', 'votes': {'positive': 0, 'abuse': 0, 'negative': 0}, 'html': '#malware #Smoke Loader<br /><strong>VT Collection:</strong> https://www.virustotal.com/gui/collection/3e3a2e51f5f354243f87aa3579e7b0d7206c9f8ce3558d9a1b5f8e6fc3fad7da<br /><strong>Reported in:</strong><br />Hatching Triage: https://tria.ge/221104-y1j46safd6<br />Intezer: https://analyze.intezer.com/analyses/4cea82d5-cc21-4fcf-b746-1802f4b31cb0<br />Hybrid Analysis: https://www.hybrid-analysis.com/sample/d1cdab058056e0e4cbf2a08851d493d9f46d1d36e65f7b284d2ecc3558e80660<br />JOESandbox: https://www.joesandbox.com/analysis/738404/0/html<br />FileScan.IO: https://www.filescan.io/reports/d1cdab058056e0e4cbf2a08851d493d9f46d1d36e65f7b284d2ecc3558e80660/d7b6b626-e2b8-4a97-8ba0-f3fa7e788687/overview<br />YOMI: https://yomi.yoroi.company/report/63657265317bd2f2ac8febfd/63657265317bd2f2ac8febfe/overview<br />malwares: https://www.malwares.com/report/file?hash=d1cdab058056e0e4cbf2a08851d493d9f46d1d36e65f7b284d2ecc3558e80660', 'tags': ['malware', 'smoke']}, 'type': 'comment', 'id': 'f-d1cdab058056e0e4cbf2a08851d493d9f46d1d36e65f7b284d2ecc3558e80660-616c6fe5', 'links': {'self': 'https://www.virustotal.com/api/v3/comments/f-d1cdab058056e0e4cbf2a08851d493d9f46d1d36e65f7b284d2ecc3558e80660-616c6fe5'}}], 'links': {'self': 'https://www.virustotal.com/api/v3/search?query=malware', 'next': 'https://www.virustotal.com/api/v3/search?cursor=CtIBChEKBGRhdGUSCQj956juzpX7AhK4AWoRc352aXJ1c3RvdGFsY2xvdWRyogELEgZTYW1wbGUiQGQxY2RhYjA1ODA1NmUwZTRjYmYyYTA4ODUxZDQ5M2Q5ZjQ2ZDFkMzZlNjVmN2IyODRkMmVjYzM1NThlODA2NjAMCxIHQ29tbWVudCJJZDFjZGFiMDU4MDU2ZTBlNGNiZjJhMDg4NTFkNDkzZDlmNDZkMWQzNmU2NWY3YjI4NGQyZWNjMzU1OGU4MDY2MC02MTZjNmZlNQwYACAB&query=malware'}}{
```

Comment:
What is this???
It looks like the JSON output from VT but the first section is odd.
If this is VT output we should name the file better. If you uploaded it to discuss what the heck it is, all good on the filename. 👍